docs/source/capi/index.rst

```diff
@@ -73,6 +73,17 @@ Functions
    :return: Return status (negative if an error occurred)
    :rtype: int
 
+.. c:function:: int openmc_cell_get_temperature(int32_t index, double* T, const int32_t* instance)
```

Comment:
I would move `instance` before `T` since it is an [in] parameter and T is an [out] parameter

Reply:
Makes sense. I just liked the symmetry with the `openmc_cell_set_temperature` method.